releasenotes/notes/apm-fix-sql-parsing-of-sqlserver-identifiers-in-square-brackets-8c625935e0bebbf7.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    APM: Fix parsing of sqlserver identifiers enclosed in square brackets.
```

Comment:
```suggestion
    APM: Fix parsing of SQL Server identifiers enclosed in square brackets.
```

Reply:
Thanks! I updated the text in the [last commit](https://github.com/DataDog/datadog-agent/pull/15019/commits/4b8331555f4e998a6171a53ca6dbdb61bcf1379e) although it's not really obvious because I also had to fix the release notes filename being too long so it's a different file now.